packages/web-components/src/components/masthead/masthead-global-bar.ts

```diff
@@ -20,17 +24,80 @@ const { stablePrefix: ddsPrefix } = ddsSettings;
  * @element dds-masthead-global-bar
  */
 @customElement(`${ddsPrefix}-masthead-global-bar`)
-class DDSMastheadGlobalBar extends FocusMixin(LitElement) {
+class DDSMastheadGlobalBar extends FocusMixin(HostListenerMixin(LitElement)) {
+  /**
+   * Media query listener handler.
+   */
+  private _mqHandler;
```

Comment:
```suggestion
  private _mqHandler: Handle | null = null;
```

Also please add `import Handle from '../../globals/internal/handle';`.